MovieAppPetProject/MovieAppPetProject/SearchMovies/SearchMovies-ViewController.swift

```diff
@@ -6,56 +6,62 @@
 //
 
 import UIKit
-import SDWebImage
 
 class SearchMoviesViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
-    
-    
+
```

Comment:
look at previous comment about strucure of IOOutlets then vars and marks and making things private 😄